[2024-05-10T06:01:00.643+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-10T06:01:00.768+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Proyecto_Nasa.enviar_mensaje scheduled__2023-11-26T00:00:00+00:00 [queued]>
[2024-05-10T06:01:00.774+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Proyecto_Nasa.enviar_mensaje scheduled__2023-11-26T00:00:00+00:00 [queued]>
[2024-05-10T06:01:00.776+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-10T06:01:00.786+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): enviar_mensaje> on 2023-11-26 00:00:00+00:00
[2024-05-10T06:01:00.792+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=988) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-10T06:01:00.794+0000] {standard_task_runner.py:63} INFO - Started process 990 to run task
[2024-05-10T06:01:00.796+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Proyecto_Nasa', 'enviar_mensaje', 'scheduled__2023-11-26T00:00:00+00:00', '--job-id', '1107', '--raw', '--subdir', 'DAGS_FOLDER/Proyecto_SpaceX_v1.py', '--cfg-path', '/tmp/tmp6x0a_nic']
[2024-05-10T06:01:00.798+0000] {standard_task_runner.py:91} INFO - Job 1107: Subtask enviar_mensaje
[2024-05-10T06:01:00.837+0000] {task_command.py:426} INFO - Running <TaskInstance: Proyecto_Nasa.enviar_mensaje scheduled__2023-11-26T00:00:00+00:00 [running]> on host 1b7f0b10dd99
[2024-05-10T06:01:00.906+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Proyecto_Nasa' AIRFLOW_CTX_TASK_ID='enviar_mensaje' AIRFLOW_CTX_EXECUTION_DATE='2023-11-26T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-26T00:00:00+00:00'
[2024-05-10T06:01:00.907+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-10T06:01:00.918+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-10T06:01:00.919+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', "echo '2023-11-26: Los datos fueron cargados'"]
[2024-05-10T06:01:00.928+0000] {subprocess.py:86} INFO - Output:
[2024-05-10T06:01:00.929+0000] {subprocess.py:93} INFO - 2023-11-26: Los datos fueron cargados
[2024-05-10T06:01:00.930+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-10T06:01:00.931+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-10T06:01:00.950+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Proyecto_Nasa, task_id=enviar_mensaje, run_id=scheduled__2023-11-26T00:00:00+00:00, execution_date=20231126T000000, start_date=20240510T060100, end_date=20240510T060100
[2024-05-10T06:01:00.969+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-10T06:01:00.981+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-10T06:01:00.984+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
